css/src/assets/styles/cool.css

```diff
@@ -0,0 +1,192 @@
+html, body, div, p, footer, section{
+  padding: 0;
+  margin: 0;
+  border: none;
+  cursor:pointer;
+  font-family: 'Roboto Sans', sans-serif;
+  font-size: inherit;
+  color: #C3B7B5
+}
+
+.group:after {
+  content:" ";
+  display: block;
+  clear: both;
+
+}
+html {
+  background:  #C3B7B5;
+  min-width: 450px
+}
+
+body {
+
+  z-index: -2;
+  background: white;
+  width: 90%;
+  margin: 20px auto;
+  border-radius: 15px;
+}
+
+.frontpage {
+
+  display: block;
+  margin: 0 auto 10px auto;
+  padding: 20px 0;
+  height: 440px;
+  background: white;
+  position: relative;
+  border-radius: 15px;
+}
+
+
+#loose {
+  width: 95%;
+  display: block;
+  margin:  auto;
+  padding: 0 2.5% 0 2.5% ;
+  height:450px;
+  float: right;
+  position: absolute;
+}
+
+.heart {
+  z-index: 2;
+  float: right;
+  position: absolute;
+  top: 100px;
+  right: 100px;
+  height: 50px;
+  width: 50px;
+  border-radius: 50%;
+  padding: 10px;
+
+}
+.emptyheart {
+  background: white;
+}
+.fullheart {
+  background: red;
+}
+
+.off {
+  display: none;
+}
+.circle {
+  z-index: 1;
+  float: right;
+  position: absolute;
+  top: 102px;
+  right: 102px;
+  height: 46px;
+  width: 46px;
+  border-radius: 50%;
+  border: 10px solid red;
+  background: transparent;
+}
+
+.on {
+  top: 70px;
+  right: 70px;
+  height: 110px;
+  width: 110px;
+  border: 10px solid transparent;
+  transition: top 1s, right 1s, height 1s, width 1s, border 1s, ease;
+}
+
+
+
+img {
```

Comment:
Styling HTML element directly many times leads to having to override this styles. I'd recommend using a `class` instead.